core/backend/src/test/hubaccess/Rebase.test.ts

```diff
@@ -0,0 +1,660 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import * as chai from "chai";
+import * as chaiAsPromised from "chai-as-promised";
+import { HubWrappers, IModelTestUtils, KnownTestLocations } from "..";
+import { BriefcaseDb, BriefcaseManager, ChannelControl, DrawingCategory, IModelHost, SqliteChangesetReader, TxnProps } from "../../core-backend";
+import { HubMock } from "../../internal/HubMock";
+import { Suite } from "mocha";
+import { Code, IModel, SubCategoryAppearance } from "@itwin/core-common";
+import { Guid, Id64String } from "@itwin/core-bentley";
+import { StashManager } from "../../StashManager";
+chai.use(chaiAsPromised);
+
+class TestIModel {
+  public iModelId: Id64String = "";
+  public drawingModelId: Id64String = "";
+  public drawingCategoryId: Id64String = "";
+  public briefcases: BriefcaseDb[] = [];
+  private _data = 0;
+  public constructor() { }
+  public async startup() {
+    HubMock.startup("TestIModel", KnownTestLocations.outputDir);
+    this.iModelId = await HubMock.createNewIModel({ iTwinId: HubMock.iTwinId, iModelName: "Test", description: "TestSubject" });
+    const b1 = await HubWrappers.downloadAndOpenBriefcase({ iTwinId: HubMock.iTwinId, iModelId: this.iModelId });
+    b1.channels.addAllowedChannel(ChannelControl.sharedChannelName);
+    b1.saveChanges();
+    const schema1 = `<?xml version="1.0" encoding="UTF-8"?>
+    <ECSchema schemaName="TestDomain" alias="ts" version="01.00" xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.1">
```

Comment:
Use EC 3.2 spec for new test schemas unless there is a specific reason why we need to use 3.1

Reply:
fixed